src/aioswitcher/api/__init__.py

```diff
@@ -556,11 +602,21 @@ class SwitcherType2Api(SwitcherApi):
     """
 
     def __init__(
-        self, device_type: DeviceType, ip_address: str, device_id: str, device_key: str
+        self,
+        device_type: DeviceType,
+        ip_address: str,
+        device_id: str,
+        device_key: str,
+        token: Union[str, None] = None,
```

Comment:
If the token default value is None and it's the last argument in the signature, why do all examples not require a token to pass an empty string?

Reply:
Not sure I got what you mean..

Reply:
In the README, you added the following example use cases:
```
asyncio.run(control_runner(DeviceType.RUNNER, "111.222.11.22", "ab1c2d", "00", ""))
asyncio.run(control_runner(DeviceType.RUNNER_MINI, "111.222.11.22", "ab1c2d", "00", ""))
asyncio.run(control_runner(DeviceType.RUNNER_S11, "111.222.11.22", "ab1c2d", "00", "zvVvd7JxtN7CgvkD1Psujw=="))
```

We are missing the concept of the default value. If you change the following two statements, the "token" value will be None, per the default value assigned. Passing an empty string is unnecessary when no token is required.

```
asyncio.run(control_runner(DeviceType.RUNNER, "111.222.11.22", "ab1c2d", "00"))
asyncio.run(control_runner(DeviceType.RUNNER_MINI, "111.222.11.22", "ab1c2d", "00"))
```

Reply:
@TomerFi Can it be resolved?